src/main/scala/com/lightbend/kafkalagexporter/InfluxDBPusherSink.scala

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package com.lightbend.kafkalagexporter
+
+import com.lightbend.kafkalagexporter.MetricsSink._
+import com.lightbend.kafkalagexporter.EndpointSink.ClusterGlobalLabels
+import org.influxdb.{InfluxDB, InfluxDBFactory, BatchOptions}
+import org.influxdb.InfluxDB.ConsistencyLevel
+import org.influxdb.dto.{Query, Point, BatchPoints}
+import java.io.IOException
+import java.util.function.Consumer
+import java.util.function.BiConsumer
+import org.influxdb.dto.QueryResult
+import java.lang.Iterable
+import com.typesafe.scalalogging.Logger
+
+import scala.util.Try
+
+object InfluxDBPusherSink
+{
+  def apply(sinkConfig: InfluxDBPusherSinkConfig, clusterGlobalLabels: ClusterGlobalLabels): MetricsSink =
+    {
+    Try(new InfluxDBPusherSink(sinkConfig, clusterGlobalLabels))
+      .fold(t => throw new IOException("Could not create Influx DB Pusher Sink", t), sink => sink)
+  }
+}
+
+class InfluxDBPusherSink private(sinkConfig: InfluxDBPusherSinkConfig, clusterGlobalLabels: ClusterGlobalLabels) extends EndpointSink(clusterGlobalLabels) {
+
+  val logger = Logger("InfluxDBPusherSink")
+  val influxDB = connect()
+  createDatabase()
```

Comment:
What happens if the "database" already exists?  Is this db created remotely?

Reply:
As per the InfluxDB API [reference here](https://docs.influxdata.com/influxdb/v1.6/query_language/database_management/#create-database):

>  if you attempt to create a database that already exists, InfluxDB does nothing and does not return an error.